DEVELOPER_GUIDE.md

```diff
@@ -174,11 +174,11 @@ Look for tar ball in /build/distributions. To run the artifact i.e. tar ball, ru
 ```
 tar -xvf opensearch-sdk-1.0.0-SNAPSHOT.tar
 ```
-TODO https://github.com/opensearch-project/opensearch-sdk/issues/52 
-Navigate to opensearch-sdk/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/ 
+TODO https://github.com/opensearch-project/opensearch-sdk-java/issues/52 
+Navigate to opensearch-sdk-java/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/ 
 - Check if src folder exists in using `ls`.
 - If the src folder does not exist, create it using `mkdir src && cd src && mkdir test && cd test && mkdir resources && cd resources`. 
-  The above command will generate a path opensearch-sdk/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/src/test/resources
+  The above command will generate a path opensearch-sdk-java/build/distributions/opensearch-sdk-1.0.0-SNAPSHOT/src/test/resources
 - Manually create a file titled `extension.yml` within the resources directory using an IDE or an in-line text editor. Below is the sample of extension.yml
```

Comment:
Note that my PR #74 changes this `extensionName` to `sample-extension` and add a note that it must exactly match the name defined in `extensions.yml`.   Please consider incorporating all my changes to this file in #74 to this PR to avoid merge conflicts, as whichever one is merged first will disrupt the other.